flepimop/gempyor_pkg/src/gempyor/config_validator.py

```diff
@@ -70,19 +70,31 @@ def validate_seedingfile(cls, values):
         seeding_file_type = values.get('seeding_file_type')        
         seeding_file = values.get('seeding_file')        
         if method == 'PoissonDistributed' and not lambda_file:
-            raise ValueError(f'Error in Seeding: A lambda_file is required when method is {method}')
+            raise ValueError(
+                f'Error in Seeding: A lambda_file is required when method is {method} '
+                f'Current value: {lambda_file}.'
+            )
         if method == 'FolderDraw' and not seeding_file_type:
-            raise ValueError('Error in Seeding: A seeding_file_type is required when method is FolderDraw')
+            raise ValueError(
+                f'Error in Seeding: A seeding_file_type is required when method is FolderDraw'
+                f'Current value: {seeding_file_type}.'
+            )
         if method == 'FromFile' and not seeding_file:
-            raise ValueError('Error in Seeding: A seeding_file is required when method is FromFile')
+            raise ValueError(
+                f'Error in Seeding: A seeding_file is required when method is FromFile '
+                f'Current value: {seeding_file}.'
+            )
         return values
     
     @model_validator(mode='before')
     def plugin_filecheck(cls, values):
         method = values.get('method')
         plugin_file_path = values.get('plugin_file_path')   
         if method == 'plugin' and not plugin_file_path:
-            raise ValueError('Error in Seeding: a plugin file path is required when method is plugin')
+            raise ValueError(
+                f'Error in Seeding: a plugin file path is required when method is plugin'
+                f'Current value: {plugin_file_path!r}. Please specify the path to the plugin file.'
+            )
```

Comment:
This and last several errors all represent the same logical arrangement: this-method-require-this argument - possible to extract that logic?